### ISSUE TITLE
Add footer with mobile responsiveness

### ISSUE THREAD

**Issue by lpandath** [tuwien-csd/damap-frontend#299]
**Changes**
Enhance the side navigation -  footer: showing icons in the mobile view and text in the desktop view.

**Additional notes**
List any notes you collected that support the original task and include a screenshot if applicable.

**Assumptions**
List any assumptions that you are aware of but still need to be reviewed.

**Acceptance Criteria**
Update the side navigation to display only icons when viewed on mobile or small devices, while retaining the full text in the desktop view.
Add a responsive footer to provide useful links or information across all devices.

#### Icon update
Due to an update in the icon set, the usage of outline icons has to be changed. For details see here: https://github.com/tuwien-csd/damap-frontend/pull/302#pullrequestreview-2275869757
TLDR;
`<mat-icon> icon_name_outline</mat-icon>`
should be changed to
`<mat-icon class="outline">icon_name</mat-icon>`

**Risks? (optional)**
Potential risks and how to best mitigate them.
